### DISCUSSION TITLE
Locked out.

### DISCUSSION THREAD

**Discussion by Travis-Ecc** [HomeITAdmin/nextcloud_geoblocker#85] [Q&A]
I ran an experiment testing the product.  I believed I had set to my Country code but it is set to AA.  I can no longer log in and administrer my Nextcloud.
Is there a way to shut this application down or disable it through the terminal or OCC commands?

**Comment by HomeITAdmin** [comment]
Hi,

please have a look into the Readme.md (https://github.com/HomeITAdmin/nextcloud_geoblocker#i-cannot-login-anymore) in the section "I cannot login anymore", everything is explained there.
